.github/workflows/docker_macos.yml

```diff
@@ -28,4 +24,30 @@ jobs:
           docker-compose up -d
           ./scripts/connect_wallet.rb
         env:
-          NETWORK: ${{ github.event.inputs.network || 'mainnet' }}
+          NETWORK: preprod
+  report:
+    needs: [build]
+    if: always()
+    runs-on: ubuntu-latest
+    steps:
+    - name: Slack Notification on failure
```

Comment:
wonder if we cannot extract this Slack part in separate file and reuse in the several places ...